### PR TITLE
Azure build pipelines definitions

### PR DESCRIPTION
**Pull request by mathieu-benoit**
Azure build pipelines definitions

### CODE REVIEW

**Comment by mathieu-benoit** [comment]
FYI @sabbour, ready for your review/merge please. Thanks!

**Comment by sabbour** [comment]
Thanks a lot for that one @mathieu-benoit! Is it also possible to add a note in the README.md to describe what the pipelines do and/or how to use them?

File: captainkube/azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+- task: HelmInstaller@0
+  displayName: 'Install Helm 2.11.0'
+  inputs:
+    helmVersion: 2.11.0
+
+- bash: helm init --client-only --wait
+  displayName: 'helm init'
+
+- bash: helm package --version $(build.buildId) --destination $(build.artifactStagingDirectory) $(system.defaultWorkingDirectory)/$(projectName)/charts/$(projectName)
```
Comment (sabbour) [inline captainkube/azure-pipelines.yml:33]: I'm curious how the --version would work here. As far as I know, this expects a `semver` while `build.buildId` doesn't generate a semVer.

Comment (mathieu-benoit) [reply captainkube/azure-pipelines.yml:33]: Good question. It looks like usually it's semver but you could set the version as a non semver too.
```
az acr helm list -n  acr-name -o table
NAME       CHART VERSION    APP VERSION    DESCRIPTION
---------  ---------------  -------------  ------------------------------------------------------------
parrot     564                             Parrot is Captain Kube's sidekick, who shows everything t...
```

So it's easy to change `--version $(build.buildId)` by `--version 1.0.$(build.buildId)`.

**Comment by mathieu-benoit** [comment]
Hi @sabbour, really good point about the Azure DevOps note. I just added one on the README.md file. Let me know what do you think. My intent is, when my blog post I told you about Azure DevOps will be published, I will reference it here. It will act as a documentation to implement a full CI/CD pipeline per app. Let me know your feedback on this. Thanks.

**Pull request merged**